### DISCUSSION TITLE
Adding device_info

### DISCUSSION THREAD

**Discussion by disforw** [kdeyev/eyeonwater#51] [Ideas]
Check out the new additions to device_info:

![IMG_3462](https://github.com/kdeyev/eyeonwater/assets/24241104/df373c44-34b3-4aa4-8092-9436a09b51a6)

**Comment by disforw** [comment]
I added `customer_name` as “manufacturer” because it’s just indicated with the word “by …”.

**Reply by kdeyev** [reply]
Oh. Cool!
